### PR TITLE
Showing usage of java servlets.

### PR DESCRIPTION
**Pull request by StanimirIliev**
Implementing a servlet which open that web page which user is selected

### CODE REVIEW

**Comment by StanimirIliev** [comment]
I made changes here

**Comment by StanimirIliev** [comment]
i made changes here

File: radio-buttons/src/main/kotlin/com/clouway/radiobuttons/IndexServlet.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class IndexServlet : HttpServlet() {
+
+    private val doc = IndexServlet::class.java.getResourceAsStream("index.html")
+            .readBytes().toString(Charset.defaultCharset())
+
+    override fun doGet(req: HttpServletRequest, resp: HttpServletResponse) {
+        resp.contentType = "text/html"
+        val out = resp.writer
+        if (req.getParameter("pageName") !== null) {
```
Comment (mgenov) [inline radio-buttons/src/main/kotlin/com/clouway/radiobuttons/IndexServlet.kt]: This page acts as controller but it's named IndexServlet.

File: radio-buttons/src/main/kotlin/com/clouway/radiobuttons/ControllerServlet.kt [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package com.clouway.radiobuttons
+
+import java.nio.charset.Charset
+import javax.servlet.http.HttpServlet
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+class ControllerServlet: HttpServlet() {
```
Comment (mgenov) [inline radio-buttons/src/main/kotlin/com/clouway/radiobuttons/ControllerServlet.kt]: This page acts as view but it's named ControllerServlet.

**Comment by StanimirIliev** [comment]
I made changes

**Pull request merged**